content/index.html

```diff
@@ -14,30 +14,44 @@ <h1>Units of scale</h1>
 
 <div class="container">
   <div class="row">
-    <div class="col-md-3 col-sm-6 col-xs-12 feature-item">
+    <div class="col-md-4 col-sm-6 col-xs-12 feature-item">
       <a href="/docs/introduction/overview/">
         <h2><i class="fa fa-sitemap"></i> Xmidt</h2>
         <p>Xmidt provides the core routing fabric between CPEs and cloud.</p>
       </a>
     </div>
-    <div class="col-md-3 col-sm-6 col-xs-12 feature-item">
+    <div class="col-md-4 col-sm-6 col-xs-12 feature-item">
       <a href="/docs/webpa/overview/">
-        <h2><i class="fas fa-search"></i> Webpa</h2>
+        <h2><i class="fas fa-exchange-alt"></i> Webpa</h2>
         <p>Webpa provides easy access to TR-181 objects via the cloud.</p>
       </a>
     </div>
-    <div class="col-md-3 col-sm-6 col-xs-12 feature-item">
+    <div class="col-md-4 col-sm-6 col-xs-12 feature-item">
       <a href="/docs/codex/overview/">
         <h2><i class="fas fa-book"></i> Codex</h2>
         <p>Codex provides a time based view into the experience of the CPE.</p>
       </a>
     </div>
-    <div class="col-md-3 col-sm-6 col-xs-12 feature-item">
+  </div>
+  <div class="row">
+    <div class="col-md-4 col-sm-6 col-xs-12 feature-item">
       <a href="/docs/wrp/overview/">
         <h2><i class="fas fa-envelope"></i> WRP</h2>
         <p>The Web Routing Protocol provides a cross transport data envelope.</p>
       </a>
     </div>
+    <div class="col-md-4 col-sm-6 col-xs-12 feature-item">
+      <a href="https://discussion.xmidt.io">
+				<h2><i class="fas fa-comments"></i> Collaborate</h2>
+        <p>We've added a place for discussing goals, issues and related interests.</p>
```

Comment:
"We've added" sounds good now, but in a few months when it isn't so new are we going to change this? "Added" kind of implies that it just happened.  Can we get rid of the first two words? Or, say something welcoming like "Come talk with us about XMiDT's goals, issues, and related interests!"

Reply:
I missed the "We've added" part.  Thank you for suggesting something better below.